packages/SwingSet/src/kernel/vat-loader/manager-subprocess-xsnap.js

```diff
@@ -268,9 +308,10 @@ export function makeXsSubprocessFactory({
             snapshotDescription,
           },
         }),
-        snapStore.saveSnapshot(vatID, snapPos, snapStoreSaveStream),
+        saveSnapshot(vatID, snapPos, snapStoreSaveStream),
       ]);
       await closeP;
+      uncompressedSizeLoaded = snapshotResults.uncompressedSize;
```

Comment:
Might as well update `uncompressedSizeLoaded` right after replacing `worker`.
```suggestion
      uncompressedSizeLoaded = snapshotResults.uncompressedSize;
      await closeP;
```